<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorChoiceWindow</class>
 <widget class="QWidget" name="ColorChoiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>265</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Color Choice</string>
  </property>
  <widget class="QSlider" name="verticalSliderRed">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderGreen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderBlue">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGreen">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBlue">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
